TestApp/app/components/PropertiesConfiguratorView.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { View, Text, TextInput, Image, FlatList, TouchableOpacity, StyleSheet } from 'react-native';
+
+import AddIcon from '../assets/add.png';
+import RemoveIcon from '../assets/remove.png';
+
+export default class PropertiesConfiguratorView extends Component {
+  render() {
+    return (
+      <View style={{ backgroundColor: 'white', padding: 10, flexDirection: 'column' }}>
+        <TouchableOpacity onPress={() => { if (this.props.allowChanges) this.props.onPropertyAdded(); }}>
+          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+            <Image style={{ height: 24, width: 24 }} source={AddIcon} />
+            <Divider />
+            <Text>Add property</Text>
+          </View>
+        </TouchableOpacity>
+        <FlatList
+          style={{ marginTop: 10 }}
```

Comment:
Why do you use some styles from the `StyleSheet` but some ones just inline? Let's use it uniformly.